mappings/net/minecraft/command/CommandBase.mapping

```diff
@@ -55,9 +55,11 @@ CLASS j net/minecraft/command/CommandBase
 		ARG 0 args
 		ARG 1 values
 	METHOD b getAliases ()Ljava/util/List;
+	METHOD b (DLjava/lang/String;IIZ)D
```

Comment:
Naming something, then clearing the name again will do that I think. @Thog Fix that!